tests/monitoring_test.go

```diff
@@ -271,22 +271,27 @@ var _ = Describe("Prometheus Alerts", func() {
 			return vmName
 		}
 
-		It("[test_id:TODO] Should not create kubevirt_ssp_vm_rbd_volume when not using DataVolume or PVC", func() {
-			vmName := createResources(false, true)
-			seriesShouldNotBeDetected(fmt.Sprintf("kubevirt_ssp_vm_rbd_volume{name='%s'}", vmName))
-			alertShouldNotBeActive("VirtualMachineCRCErrors")
-		})
-
 		It("[test_id:TODO] Should not fire VirtualMachineCRCErrors when rxbounce is enabled", func() {
 			vmName := createResources(true, true)
-			waitForSeriesToBeDetected(fmt.Sprintf("kubevirt_ssp_vm_rbd_volume{name='%s', rxbounce_enabled='true'}", vmName))
+			waitForSeriesToBeDetected(fmt.Sprintf("kubevirt_ssp_vm_rbd_block_volume_without_rxbounce{name='%s'}", vmName))
 			alertShouldNotBeActive("VirtualMachineCRCErrors")
 		})
 
 		It("[test_id:10549] Should fire VirtualMachineCRCErrors when rxbounce is disabled", func() {
 			vmName := createResources(true, false)
-			waitForSeriesToBeDetected(fmt.Sprintf("kubevirt_ssp_vm_rbd_volume{name='%s', rxbounce_enabled='false'}", vmName))
+			waitForSeriesToBeDetected(fmt.Sprintf("kubevirt_ssp_vm_rbd_block_volume_without_rxbounce{name='%s'}", vmName))
 			waitForAlertToActivate("VirtualMachineCRCErrors")
+
+			err := apiClient.Delete(ctx, vm)
+			Expect(err).ToNot(HaveOccurred())
+
+			Eventually(func(g Gomega) {
+				err := apiClient.Get(ctx, types.NamespacedName{Name: vmName, Namespace: strategy.GetNamespace()}, vm)
+				g.Expect(err).To(HaveOccurred())
```

Comment:
You don't need this line any more, it's covered by `MatchError` already.

Reply:
removed